src/components/TextPopup/TextPopup.js

```diff
@@ -61,9 +69,19 @@ const TextPopup = () => {
       data-element="textPopup"
       ref={popupRef}
       style={{ ...position }}
-      onClick={() => dispatch(actions.closeElement('textPopup'))}
+      onClick={onClose}
     >
       <CustomizablePopup dataElement="textPopup">
+        {fullAPI && (
+          <ActionButton
+            dataElement="editTextButton"
+            title="action.edit"
+            img="ic_edit_black_24px"
+            onClick={() =>
+              dispatch(actions.openElement('editTextModal'))
```

Comment:
Wrap this onClick to `useCallback` so it doesn't cause re-render of ActionButton on every render